enable: true
action_server_name: "/move_base" #/task_supervisor
plan_topic: "/move_base/NavfnROS/plan" #"/move_base/DWAPlannerROS/local_plan" #"/move_base/NavfnROS/plan"
costmap_topic: "/move_base/local_costmap/costmap"
cmd_vel_topic: "/cmd_vel_mux/teleop/keyboard"
obstruction_threshold: 85
clearing_timeout: 5.0 # sec, if plan is still obstructed after this period, abort action
control_frequency: 20.0 # Hz, override will be sent in this frequency
